### ISSUE TITLE
Can't locate Text/ANSITable/BorderStyle/ASCII.pm in @INC

### ISSUE THREAD

**Issue by eserte** [perlancar/perl-Data-Format-Pretty-Console#1]
t/01-basics.t in Data-Format-Pretty-Console-0.390 may fail, probably due to missing prereqs:
```
Can't locate Text/ANSITable/BorderStyle/ASCII.pm in @INC (you may need to install the Text::ANSITable::BorderStyle::ASCII module) ...
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 17.
t/01-basics.t ............ 
Dubious, test returned 2 (wstat 512, 0x200)
All 17 subtests passed 
```

**Comment by perlpunk** [comment]
We have a similar error on openSUSE:
```
[   14s] load_module_with_optional_args(): Failed to load module 'ASCII::SingleLine' (all prefixes tried: BorderStyle) at /usr/lib/perl5/vendor_perl/5.32.1/Module/Load/Util.pm line 71.
[   14s] # Tests were run but no plan was declared and done_testing() was not seen.
[   14s] # Looks like your test exited with 25 just after 20.
[   14s] t/01-basics.t ............ 
[   14s] Dubious, test returned 25 (wstat 6400, 0x1900)
[   14s] All 20 subtests passed 
```
https://build.opensuse.org/package/live_build_log/devel:languages:perl:autoupdate/perl-Data-Format-Pretty-Console/standard/x86_64

**Comment by perlancar** [comment]
Fixed in da02967
